packages/sources/coinbase-prime/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { config } from './config'
+import { balance } from './endpoint'
+
+export const adapter = new Adapter({
+  defaultEndpoint: balance.name,
+  name: 'COINBASE-PRIME',
+  config,
+  endpoints: [balance],
+  rateLimiting: {
+    tiers: {
+      default: {
+        rateLimit1s: 25,
```

Comment:
feels maybe a little overkill, any idea how often we'd expect updates on the coinbase side?

Reply:
It probably is for a single asset in a single portfolio but we usually configure whatever the data provider's doc mention. If for any reason in the future there are requests for other assets or portfolios, all of the requests would share this rate limit so this helps future-proof the adapter too.